instrumentation/runtime-telemetry/runtime-telemetry-java17/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java17/internal/DurationUtil.java

```diff
@@ -12,13 +12,12 @@
  * any time.
  */
 public final class DurationUtil {
-  private static final double NANOS_PER_MILLI = 1e6;
+  private static final double NANOS_PER_SECOND = 1e9;
```

Comment:
elsewhere we use `TimeUnit.SECONDS.toNanos(1)`

Reply:
Yes, that is correct, I kept this scientific notation because it was already there. It may be a good idea to apply unified approach, however it would be even better to define it in a single place. I'll take a look at this. Thanks for the feedback.